docs/stress-test.md

```diff
@@ -122,3 +141,128 @@ platform is deployment-specific but the following are some examples of dashboard
   memory usage remains high even after the purge starts because the hub aggressively caches DB state in
   memory and is apparently not cleaning up the cache references even after spawners and users are deleted
   from the database.
+
+<a name="profiling"></a>
+## Profiling
```

Comment:
What do you think about splitting this out into its own doc, i.e. docs/profiling.md? It works in here as well, but profiling is something that could be useful in normal usage of the hub rather than just a stress test scale up / steady state run as well.

Reply:
I like that idea. I'll get it moved.